entropylab/api/experiment.py

```diff
@@ -158,6 +159,7 @@ def __init__(
         self.label = label
         self.story = story
         self._kwargs = {}
+        self._user = user
```

Comment:
`user` might be `None`. Maybe check if it is and put `""` into `self._user` in that case?